Test needs an update to strip `.drive` from the path before checking no `:`:.

```
FAILED tests/test_s3_specific.py::test_mrap_local_path_windows_encoding - AssertionError: Colon found in local path on simulated Windows: C:\Users\RUNNER~1\AppData\Local\Temp\tmpc_bmszu4\arn%3Aaws%3As3%3A%3A123456789012%3Aaccesspoint\my-mrap.mrap\some\key.txt
assert ':' not in 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\tmpc_bmszu4\\arn%3Aaws%3As3%3A%3A123456789012%3Aaccesspoint\\my-mrap.mrap\\some\\key.txt'
```

--- a/tests/test_s3_specific.py
+++ b/tests/test_s3_specific.py
@@ -405,7 +405,5 @@ def test_mrap_local_path_windows_encoding(monkeypatch, s3_rig):
     p = client.CloudPath(f"s3://{_MRAP_ARN}/some/key.txt")
     # strip drive (e.g. "C:") since it legitimately contains a colon on Windows
     local_no_drive = str(p._local)[len(p._local.drive) :]
-    assert ":" not in local_no_drive, (
-        f"Colon found in local path on simulated Windows: {p._local}"
-    )
+    assert ":" not in local_no_drive, f"Colon found in local path on simulated Windows: {p._local}"
     assert "%3A" in local_no_drive

--- a/tests/test_s3_specific.py
+++ b/tests/test_s3_specific.py
@@ -403,5 +403,9 @@ def test_mrap_local_path_windows_encoding(monkeypatch, s3_rig):
     monkeypatch.setattr(s3path_module.sys, "platform", "win32")
     client = s3_rig.client_class()
     p = client.CloudPath(f"s3://{_MRAP_ARN}/some/key.txt")
-    assert ":" not in str(p._local), f"Colon found in local path on simulated Windows: {p._local}"
-    assert "%3A" in str(p._local)
+    # strip drive (e.g. "C:") since it legitimately contains a colon on Windows
+    local_no_drive = str(p._local)[len(p._local.drive) :]
+    assert ":" not in local_no_drive, (
+        f"Colon found in local path on simulated Windows: {p._local}"
+    )
+    assert "%3A" in local_no_drive

